client/my-sites/stats/stats-date-picker/index.jsx

```diff
@@ -75,8 +89,19 @@ class StatsDatePicker extends Component {
 		return formattedDate;
 	}
 
+	renderQueryDate() {
+		const { queryDate, moment, translate } = this.props;
```

Comment:
Do we need to handle a case where `queryDate` is null?  Seems unlikely to happen but curious if we should still add an early return here if so.